src/streamlist/StreamList.js

```diff
@@ -32,29 +41,36 @@ export default class StreamList extends React.Component {
   render() {
     const { streams, selected, showDescriptions, showSwitch, onNarrow, onSwitch } = this.props;
     const sortedStreams = streams.sort((a, b) => a.name.localeCompare(b.name));
+    const noResults = streams.length === 0;
+
 
     return (
-      <FlatList
-        style={styles.list}
-        initialNumToRender={sortedStreams.length}
-        data={sortedStreams}
-        keyExtractor={item => item.stream_id}
-        renderItem={({ item }) => (
-          <StreamItem
-            name={item.name}
-            iconSize={16}
-            isPrivate={item.invite_only}
-            description={(showDescriptions) ? item.description : ''}
-            color={item.color}
-            isSelected={item.name === selected}
-            isMuted={item.in_home_view === false} // if 'undefined' is not muted
-            showSwitch={showSwitch}
-            isSwitchedOn={item.subscribed}
-            onPress={onNarrow}
-            onSwitch={onSwitch}
-          />
-        )}
-      />
+      <View style={styles.container} >
+        {noResults &&
```

Comment:
Was trying to keep things consistent with how Empty state is handled in https://github.com/zulip/zulip-mobile/blob/master/src/search/SearchMessagesScreen.js.

Since all three different search results (Messages, Users, Subscriptions) are rendered in three completely different ways, wondering if creating a separate component that houses the logic of whether to show empty state or the list stored is a possibility.